downstream/assemblies/security/assembly-firewall-policy-manage.adoc

```diff
@@ -0,0 +1,43 @@
+////
+Retains the context of the parent assembly if this assembly is nested within another assembly.
+For more information about nesting assemblies, see: https://redhat-documentation.github.io/modular-docs/#nesting-assemblies
+See also the complementary step on the last line of this file.
+////
+
+ifdef::context[:parent-context: {context}]
+:imagesdir: images
+
+////
+ Base the file name and the ID on the assembly title. For example:
+* file name: assembly-my-user-story.adoc
+* ID: [id="assembly-my-user-story_{context}"]
+* Title: = My user story
+
+The ID is an anchor that links to the module. Avoid changing it after the module has been published to ensure existing links are not broken. Include {context} in the ID so the assembly can be reused.
+////
+
+[id="assembly-firewall-policy-management_{context}"]
+
+= Firewall policy management with Ansible security automation
+
+:context: firewall-policy-management
+
+[role="_abstract"]
+
+As a security operator, you can use Ansible security automation to manage multiple firewall policies. Create and delete firewall rules to block or unblock a source IP address from accessing a destination IP address.
+
+include::security/con-about-firewall-policy-management.adoc[leveloffset=+1]
+include::security/con-automating-firewall-rules.adoc[leveloffset=+1]
+
+////
+[leveloffset=+1] ensures that when a module title is a level 1 heading (= Title), the heading will be interpreted as a level-2 heading (== Title) in the assembly. Use [leveloffset=+2] and [leveloffset=+3] to nest modules in an assembly.
+////
+
+include::security/proc-creating-firewall-rule.adoc[leveloffset=+2]
+include::security/proc-deleting-firewall-rule.adoc[leveloffset=+2]
+
```

Comment:
There's no real reason to make these procedures a subsection of the automat firewall rules concept module. Consider changing  leveloffset=+1 for both.

Reply:
@samccann Thanks for the review. I changed it to leveloffset=+1.